scripts/publish.ts

```diff
@@ -0,0 +1,54 @@
+import { execa } from "execa"
+import compare from "semver/functions/compare"
+
+import contractsPkgJson from "@semaphore-protocol/contracts/package.json"
+
+const { version: contractsLocalVersion } = contractsPkgJson
+
+async function maybePushToSoldeer() {
+    // api not documented, may change, found by inspecting the network tab
+    const response = await fetch(
+        "https://api.soldeer.xyz/api/v1/revision?project_name=semaphore-protocol-contracts&limit=1"
```

Comment:
I suggest `semaphore-protocol-contracts` as pkg on soldeer. soldeer does not support orgs.

Reply:
What about `semaphore-protocol-contracts`? Sounds easier to remember